src/witan/send/population/england.clj

```diff
@@ -0,0 +1,231 @@
+  [& {:as options}]
+  (let [{::keys [ons-pop-f]
+         :keys [min-academic-year
+                max-academic-year
+                min-calendar-year
+                min-snpp-calendar-year
+                max-calendar-year]
+         :as options-with-defaults} (merge default-options options)
+        ons-pop-options (merge options-with-defaults
+                               (when min-academic-year      {:min-age       (-> min-academic-year      ncy->age)})
+                               (when max-academic-year      {:max-age       (-> max-academic-year      ncy->age)})
+                               (when min-calendar-year      {:min-year      (-> min-calendar-year      census-year->year)})
+                               (when min-snpp-calendar-year {:min-snpp-year (-> min-snpp-calendar-year census-year->year)})
+                               (when max-calendar-year      {:max-year      (-> max-calendar-year      census-year->year)}))
+        ons-pop-ds      (ons-pop-f ons-pop-options)]
+    (ons-pop-ds->witan-send-pop-ds ons-pop-ds options-with-defaults)))
+
+(comment ;; Example: Surrey NCY 7 for `calendar-year`s 2021 to 2026 using default ONS population function
+  (->dataset {:ctyuanm-f              #{"Surrey"}
+              :min-academic-year      7
+              :max-academic-year      7
+              :min-calendar-year      2021
+              :max-calendar-year      2026})
+  ;;=> witan.population from [myebtablesenglandwales/myebtablesenglandwales20112024.xlsx]MYEB1: long by CTYUA (persons) and 2022snpppopulationsyoa/2022snpppopulationsyoamigcat23/2022 SNPP Population persons.csv: long by CTYUA concatenated at min-snpp-year of 2022 [6 8]:
+  ;;   
+  ;;   | :ctyua23cd | :ctyua23nm | :year | :calendar-year | :age | :academic-year |    :sex | :population |
+  ;;   |------------|------------|------:|---------------:|-----:|---------------:|---------|------------:|
+  ;;   |  E10000030 |     Surrey |  2020 |           2021 |   11 |              7 | persons |   15052.000 |
+  ;;   |  E10000030 |     Surrey |  2021 |           2022 |   11 |              7 | persons |   15323.000 |
+  ;;   |  E10000030 |     Surrey |  2022 |           2023 |   11 |              7 | persons |   15885.000 |
+  ;;   |  E10000030 |     Surrey |  2023 |           2024 |   11 |              7 | persons |   16033.011 |
+  ;;   |  E10000030 |     Surrey |  2024 |           2025 |   11 |              7 | persons |   15533.584 |
+  ;;   |  E10000030 |     Surrey |  2025 |           2026 |   11 |              7 | persons |   15271.068 |
+  ;;   
+
+  :rcf)
+
+(def output-columns
+  "Output columns for `witan.send` population.csv file."
+  [:calendar-year :academic-year :population])
+
+(defn write!
+  "Writes required output columns from witan.send.population dataset `ds` to CSV file `file-name`."
+  [ds file-name]
+  (-> ds
+      (tc/select-columns output-columns)
+      (tc/write! file-name)))
```

Comment:
worth having an overrideable version of this to allow different output columns but defaulting to the ones in this ns?